src/uikit.cs

```diff
@@ -16687,6 +16732,15 @@ interface UIFocusItem : UIFocusEnvironment
 		[Abstract]
 		[Export ("canBecomeFocused")]
 		bool CanBecomeFocused { get; }
+
+		[TV (12, 0), iOS (12, 0), NoWatch]
+		[Abstract]
```

Comment:
This is a breaking change: can't add new `[Abstract]` members to an existing protocol.

It should go into a `#if XAMCORE_4_0` block.